### DISCUSSION TITLE
Is it possible to simulate Unitree G1 humanoid?

### DISCUSSION THREAD

**Discussion by ramkumarkoppu** [isaac-sim/IsaacLab#411] [Q&A] [answered]
Is it possible to simulate Unitree G1 humanoid with it?

**Comment by abizovnuralem** [comment]
Yes, I am doing it, but I am stuck in the local minimum of PPO. I need some deeper setup for it.

**Comment by Mayankm96** [comment, marked as answer]
This is now supported.
